pkg/ui/src/views/reports/containers/range/rangeTable.tsx

```diff
@@ -449,6 +453,7 @@ export default class RangeTable extends React.Component<RangeTableProps, {}> {
         problems: this.contentProblems(info.problems, awaitingGC),
         raftState: raftState,
         quiescent: info.quiescent ? rangeTableQuiescent : rangeTableEmptyContent,
+        ticking: this.createContent(info.ticking.toString()),
```

Comment:
I think it would be nice to roll this in with the quiescent row.  I'm imagining we'd show quiescent and empty as normal states and "quiescent but ticking" and "not ticking but not quiescent" as error states.

Reply:
Yeah, I just wanted to get something in quickly for cyan without having to fiddle with and test all the combinations.

Reply:
I'm fine with that, we can just have @BramGruneir come through and add some niceties here.